client/my-sites/plans-features-main/index.tsx

```diff
@@ -439,18 +439,19 @@ const PlansFeaturesMain = ( {
 		plans: gridPlansForFeaturesGrid.map( ( gridPlan ) => gridPlan.planSlug ),
 	};
 
-	const planActionOverrides: PlanActionOverrides | undefined = {
-		loggedInFreePlan: {
-			status: isPlanUpsellEnabledOnFreeDomain.isLoading ? 'blocked' : 'enabled',
-		},
-	};
+	/**
+	 * The effects on /plans page need to be checked if this variable is initialized
+	 */
+	let planActionOverrides: PlanActionOverrides | undefined = undefined;
```

Comment:
Would you mind elaborating on this when you have time? It sounds like there's logic in an effect in the `/plans` grid pricing component that checks whether or not `planActionOverrides` is initialized?

Reply:
This was my initial prognosis 😬 ... Have to dive a bit deeper into the behaviour to understand this.